include/ginkgo/core/matrix/dense.hpp

```diff
@@ -441,6 +441,50 @@ class Dense
         return create_submatrix(rows, columns, this->get_stride());
     }
 
+    /*
+     * Create a real view of the (potentially) complex original matrix.
```

Comment:
Can add a couple of lines saying that you are viewing the complex matrix with a `reinterpret_cast` with stride 2*original_stride ?